src/modules/audio/audioApi.ts

```diff
@@ -71,40 +75,34 @@ export const deleteLanguageVersionSeries = (
 export const fetchSearchTags = async (
   input: string,
   language: string,
-): Promise<TagSearchResult> => {
+): Promise<ITagsSearchResult> => {
   const response = await fetchAuthorized(
     `${baseUrl}/tag-search/?language=${language}&query=${input}`,
   );
   return resolveJsonOrRejectWithError(response);
 };
 
-export const fetchSeries = (
-  id: number | string,
-  language?: string,
-): Promise<PodcastSeriesApiType> => {
+export const fetchSeries = (id: number | string, language?: string): Promise<ISeries> => {
```

Comment:
Var det ikke string som knakk backend?

Reply:
![image](https://user-images.githubusercontent.com/10486712/157218534-9db817ec-bbc3-40a3-8deb-b0821e418471.png)


I don't know, you tell me :>

Reply:
Det stemmer nok, men det var du som ikke ville ha det sånn!